### PR TITLE
Update flashing doc

### PR DESCRIPTION
**Pull request by utzcoz**
Looks like the second item on issue is my mistake to enter official `recovery`, so I don't add new content for it. 

Close https://github.com/maruos/maruos/issues/173

### CODE REVIEW

File: docs/user/twrp.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ There are many ways to install TWRP, but here's one that works well.
 
         $ fastboot flash recovery <twrp filename>.img
 
+    If above command failed with no `recovery` partition error, you can try the following command:
```
Comment (pdsouza) [inline docs/user/twrp.md:39]: It looks like the official instructions suggest a two-step process for installing TWRP permanently on devices without the `recovery` partition: https://twrp.me/google/googlepixelxl.html. First `fastboot boot twrp.img` and then install a new TWRP zip installer file. But if this command works for now, I think we can keep it to keep the instructions simple.

**Review by pdsouza — APPROVED** [review]
LGTM. See my note on the two-step process for installation that maybe we can add later.

File: docs/user/twrp.md (hunk 1 above)
Comment (utzcoz) [reply docs/user/twrp.md:39]: It works fine for me with sailfish.

**Comment by utzcoz** [comment]
> LGTM. See my note on the two-step process for installation that maybe we can add later.

Thanks.

> This will temporarily boot TWRP on your device. If you are using a lockscreen pin/pattern/password and do not get prompted to enter your passord, reboot to the bootloader and try again. Go to install and browse to the zip and install the zip. The zip will install TWRP to both boot slots. Installing TWRP at this time will remove root if you are currently rooted.

I only use `fastboot flash boot twrp.img` to boot system to twrp temporarily, but it is meaningful. Maybe it needs a new commit.

**Pull request merged**